docs/modules/ROOT/pages/appendix.adoc

```diff
@@ -41,12 +41,12 @@ We do support the following processing instructions:
 * `<?assert followed by a valid precondition ?>`
 * `<?assume followed by a valid precondition ?>`
 
-Look up valid preconditions <<concepts_preconditions,here>>. The full XMl schema for <<concepts_migrations_catalog-based, catalog-based migrations>> looks like this:
+Look up valid preconditions xref:concepts.adoc#concepts_preconditions[here]. The full XMl schema for xref:concepts.adoc#concepts_migrations_catalog-based[catalog-based migrations] looks like this:
 
 .migration.xsd
 [source,xml,indent=0,tabsize=2]
 ----
-include::../neo4j-migrations-core/src/main/resources/ac/simons/neo4j/migrations/core/migration.xsd[]
+include::example$migration.xsd[]
```

Comment:
We should avoid using relative paths in Antora. Here we include an "example": https://docs.antora.org/antora/latest/page/include-an-example/